activejob/Rakefile

```diff
@@ -36,19 +36,15 @@ namespace :test do
     Rake::TestTask.new(adapter => "test:env:#{adapter}") do |t|
       t.description = "Run adapter tests for #{adapter}"
       t.libs << "test"
-      t.test_files = FileList["test/cases/**/*_test.rb"].reject {
-        |x| x.include?("delayed_job") && adapter != "delayed_job"
```

Comment:
Having this `skip?` logic here feels wrong to me. I think it should live in the test files so it's easier to follow what's going on by just reading the test file.